src/Common/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -492,5 +492,32 @@ internal static void SetCurrentTransaction(Transaction transaction)
         {
             Transaction.Current = transaction;
         }
+
+        internal static Timer CreateGlobalTimer(TimerCallback callback, object state, int dueTime, int period)
```

Comment:
In case this change is causing you to add an additional dependency in System.Data.Common, it would make sense to move the change to System.Data.SqlClient\src\System\Data\Common\AdapterUtil.SqlClient.cs

Reply:
I was assuming I could make use of it for ODBC which has the same issue? https://github.com/dotnet/corefx/pull/26066

Reply:
Yeah, the code should be shared between the two PRs.

In this case you can create a src/Common/src/System/Data/Common/AdapterUtil.Drivers.cs and add this `CreateGlobalTimer` function to a partial `ADP` class. Then you could include the reference to the file in System.Data.SqlClient and S.D.Odbc. This way the function doesn't need to be compiled into S.D.Common.

The DbConnectionPool code that is duplicated in `System/Data/ProviderBase` of S.D.SqlClient and S.D.Odbc should be moved to a single code as well. 

**A little bit of History**
What is happening in .Net framework is that S.D.Odbc and S.D.SqlClient reside in System.Data.dll 
While moving these assemblies to .Net core, the connection pool code got duplicated. If there was a single set of files for connection pool, then a single PR would have taken care of this issue for both Odbc and SqlClient. 
Consolidation of connection pool code is outside the scope of this PR. I will open an issue to consolidate the code for Connection Pool.

Reply:
Done